__tests__/suites/paypal-button-click-pom.test.ts

```diff
@@ -2,26 +2,22 @@ import { expect } from "chai";
 import { CheckoutPage } from "../pages/checkoutPage";
 import { HomePage } from "../pages/homePage";
 
-describe("Testing Paypal buttons", () => {
-  beforeEach("Open StackDemo", () => {
-    browser.url("/demo/checkout/#/pattern/client");
+describe("Testing Paypal buttons", async () => {
```

Comment:
no need for an async with describe. Can we confirm if this is hampering functionality in any way?

Reply:
done. it wont impact functionality. 
the wdio docs define it without async so we are good.
https://webdriver.io/docs/sync-vs-async/